phases/register.go

```diff
@@ -1,9 +1,104 @@
 package phases
 
-import "fmt"
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
 
-// Register ...
-func Register(progress Progress) error {
-	fmt.Println("Register")
+	"github.com/bitrise-io/go-utils/log"
+)
+
+const baseURL = "https://app.bitrise.io"
+
+func startAppRegistration() (string, error) {
+	return "", nil
+}
+
+func registerSSHKey() error {
+	return nil
+}
+
+func registerWebhook(appSlug string, apiToken string) error {
+	url := fmt.Sprintf("%s/app/%s/register-webhook.json", baseURL, appSlug)
+	request, err := http.NewRequest(http.MethodPost, url, nil)
```

Comment:
Note: In case the network request fails due to unrelialble connection, would be good to retry it.